deno2/tools/build.py

```diff
@@ -23,6 +23,9 @@ def main():
     os.chdir(root_path)
     buildName = "Debug" if args.debug else "Default"
     buildDir = os.path.join(root_path, "out", buildName)
+    # Default to non-Googler configuration.
+    if 'DEPOT_TOOLS_WIN_TOOLCHAIN' not in os.environ:
```

Comment:
In addition to doing this, also print out a me.ssage. Otherwise people are going to scratch their head too much when running gn directly and it doesn't work.

Reply:
Do you think the note I added in README.md is insufficient?